fargate/video-paper-builder.yml

```diff
@@ -121,6 +121,10 @@ Parameters:
   SchoologySecret:
     Type: String
     Description: The Schoology consumer secret
+  DebugProduction:
+    Type: String
+    Default: false
+    Description: Set to "true" to enable development style logging in production
```

Comment:
You could add:
`"AllowedValues" : ["true", "false"]`
To make this more error proof.